### DISCUSSION TITLE
Rejoining a room after being kicked in multi

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#21750] [Q&A]
You can join a multiplayer room for unlimited amount of time after being kicked every time you join. Restricting mechanism should be implemented.
For example: You can give only 1 attempt to rejoin after being kicked because someone could be kicked by an accident.

**Comment by cheeezie** [comment]
I'd agree with that, but there's already some measurements in place such as adding a password to join. But I'd understand the frustration if it's an open lobby.

**Reply by Loreos7** [reply]
I accidentally guessed someone's room password recently (because its always something simple like "123") and entered the room like 20 times in a row. Host was trying to kick me every time I joined the room. It shouldn't be like that.

**Reply by peppy** [reply]
They could change the password?

**Reply by Loreos7** [reply]
Sounds fair enough

**Comment by Loreos7** [comment]
@Joehuu Could you place tags `area:multiplayer` and `type:behavioural`, please?
Maybe other ones too? https://github.com/ppy/osu/discussions?discussions_q=is%3Aopen+author%3ALoreos7+is%3Aunanswered+

**Reply by Joehuu** [reply]
Done here.

> Maybe other ones too?

I could, but I would have to account for all discussions, not just yours. May do next time I wanna make the discussions cleaner.
